docs/source/sysadmin_guide/pbft_adding_removing_node.rst

```diff
@@ -0,0 +1,158 @@
+******************************
+Adding or Removing a PBFT Node
+******************************
+
+Membership of a Sawtooth PBFT network is controlled by the on-chain setting
+``sawtooth.consensus.pbft.members``. An administrator must update this setting
+when :ref:`adding a new node <adding-a-pbft-node-label>` or
+:ref:`removing an existing node <removing-a-pbft-node-label>`.
+
+All nodes in the network monitor this setting. When a node detects a change,
+it updates its local list of PBFT members to match the new list.
+
+.. _adding-a-pbft-node-label:
+
+Adding a PBFT Node
+==================
+
+To add a new node to an existing PBFT network, you will install and configure
+the node, start it and wait for it to catch up with the rest of the network,
+then update ``sawtooth.consensus.pbft.members`` on an existing node.
+
+You can add several nodes at the same time.
+
+1. Install and configure Sawtooth on the new node, as described in
+   :doc:`setting_up_sawtooth_poet-sim`.
+   Skip the procedure to create a genesis block.
+
+#. Start the new node. It will join the Sawtooth network and start the block
+   catch-up procedure, where it receives and commits the blocks that are already
+   on the blockchain. (For details, see the `PBFT node catch-up RFC
+   <https://github.com/hyperledger/sawtooth-rfcs/blob/master/text/0031-pbft-node-catchup.md>`_.)
+
+#. Wait for the node to catch up to the rest of the network (get within a few
+   blocks of the chain head). If the new node becomes a PBFT member before it
+   has caught up, the entire network could slow down or stall while the new
+   member node catches up.
+
+   This process can take a while, depending on the blockchain size, node's
+   hardware, and network speed. You can look at the logs or run
+   :ref:`sawtooth block list <sawtooth-block-list-label>` to check on the
+   new node's progress.
+
+#. Copy the new node's public validator key for use in the next step.
+
+   .. code-block:: console
+
+      $ cat /etc/sawtooth/keys/validator.pub
+
+   This step assumes that the validator key is stored in the default location,
+   ``/etc/sawtooth/keys``. If not, use the location specified by the ``key_dir``
+   setting (see :doc:`configuring_sawtooth/path_configuration_file`).
+
+#. On an existing member node, update ``sawtooth.consensus.pbft.members`` to
```

Comment:
I would generalize this; "Update the setting to include the public key ... Run the following steps using a key that has permission ...; that is, a public key listed in.... Usually, the public key of the node that created the genesis block is on this list...."

It's helpful to point out that it's commonly the genesis node's key that's in this list, but could be some special key that doesn't belong to any validator.

Reply:
This is a lot of information to include in this step (and doesn't really "belong" to the PBFT procedure), so I'll format the info as a separate tip.  

Ideally, the Sawtooth doc should have a separate topic to explain this info, because several procedures show how to change on-chain settings.  When that happens, this tip can be replaced with a link to the general information.